build.sbt

```diff
@@ -73,22 +85,32 @@ lazy val `rho-examples` = project
       Seq(
         exampleDeps,
         libraryDependencies ++= Seq(logbackClassic, http4sXmlInstances),
-        dontPublish
+        dontPublish,
+        scalacOptions --= versionSpecificDisabledFlags(scalaVersion.value)
       ): _*)
   .dependsOn(`rho-swagger`, `rho-hal`)
 
-lazy val compileFlags = Seq(
+lazy val compilerFlags = Seq(
   "-feature",
   "-deprecation",
   "-unchecked",
   "-language:higherKinds",
   "-language:existentials",
   "-language:implicitConversions",
   "-Ywarn-unused",
-  "-Ypartial-unification",
   "-Xfatal-warnings"
 )
 
+def versionSpecificEnabledFlags(version: String) = (CrossVersion.partialVersion(version) match {
+  case Some((2, 13)) => Seq.empty[String]
+  case _ => Seq("-Ypartial-unification")
+})
+
+def versionSpecificDisabledFlags(version: String) = (CrossVersion.partialVersion(version) match {
+  case Some((2, 13)) => Seq("-Xfatal-warnings")
```

Comment:
What is the reason for disabling fatal warnings for 2.13 ?

Reply:
There's a false-positive "unreachable code" warning in `StaticContentService`. Cause most likely one of these bugs: https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+unreachable+2.13